serverAddons/AAW/addons/AAW_Server/Tickets/fn_init.sqf

```diff
@@ -162,7 +162,7 @@ GVAR(deactivateTicketSystem) = false;
         ["ticketsChanged", {
             if (GVAR(deactivateTicketSystem)) exitWith {};
             private _dialog = uiNamespace getVariable [UIVAR(TicketStatus), displayNull];
-            if !(isNull _dialog) then {
+            if (!isNull _dialog) then {
                 (_dialog displayCtrl 2011) ctrlSetText (missionNamespace getVariable [format [QEGVAR(Common,Flag_%1), EGVAR(Common,competingSides) select 0], "#(argb,8,8,3)color(0.5,0.5,0.5,1)"]);
```

Comment:
i think this lines are to long. maybe we should try to split this kind of lines more